practice-one/pages/index.html

```diff
@@ -0,0 +1,605 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="../styles/main.css">
+  <link rel="stylesheet" href="../assets/font-awesome/css/all.css">
+</head>
+<body>
+  <!-- Start Header -->
+  <header class="header section">
+    <div class="header-container container">
+      <div class="header-nav">
+        <a href="#" class="logo"><span>Digi</span>ency</a>
```

Comment:
The logo should be an image instead of text. It's png file.
![Digiency](https://user-images.githubusercontent.com/11568347/180938609-54eee636-bba6-463f-8a27-f04674d6cac9.png)

Reply:
Thank you for your comments. I will update it